src/HotChocolate/Fusion-vnext/test/Fusion.Composition.Tests/PostMergeValidationRules/IsInvalidFieldRuleTests.cs

```diff
@@ -101,6 +101,86 @@ type Person {
                     "The @is directive on argument 'Query.personById(id:)' in schema 'A' "
                     + "specifies an invalid field selection against the composed schema."
                 ]
+            },
+            // Type of argument does not match field type.
+            {
+                [
+                    """
+                    # Schema A
+                    type Query {
+                        personById(id: Int! @is(field: "id")): Person @lookup
+                    }
+
+                    type Person {
+                        id: ID!
+                        name: String
+                    }
+                    """
+                ],
+                [
+                    "The @is directive on argument 'Query.personById(id:)' in schema 'A' "
+                    + "specifies an invalid field selection against the composed schema."
+                ]
+            },
+            // List output type, Singular input type.
+            {
+                [
+                    """
+                    # Schema A
+                    type Query {
+                        personsById(id: ID! @is(field: "id")): [Person!]! @lookup
+                    }
+
+                    type Person {
+                        id: ID!
+                        name: String
+                    }
+                    """
+                ],
+                [
+                    "The @is directive on argument 'Query.personsById(id:)' in schema 'A' "
+                    + "specifies an invalid field selection against the composed schema."
+                ]
+            },
+            // Singular output type, List input type.
+            {
+                [
+                    """
+                    # Schema A
+                    type Query {
+                        personByIds(ids: [ID!]! @is(field: "id")): Person! @lookup
+                    }
+
+                    type Person {
+                        id: ID!
+                        name: String
+                    }
+                    """
+                ],
+                [
+                    "The @is directive on argument 'Query.personByIds(ids:)' in schema 'A' "
+                    + "specifies an invalid field selection against the composed schema."
+                ]
+            },
+            // List output type, List input type (valid for Fusion v1 batch lookups).
```

Comment:
Will this be supported in the executor?

Reply:
I talked about it with Michael at some point to make the transition easier, but it will be a big change, so I'm considering it invalid at the moment. Most people in Fusion v1 weren't using the attributes so when auto-converting from v1 to v2 we can hopefully just ignore these batch fields.